content/docs/getting-started/install-github-app/index.md

```diff
@@ -14,7 +14,7 @@ If your IT department filters your outbound access by hostname, you may need to
 
 ## Step 1: Install the Roadie GitHub app
 
-Installing the GitHub App will allow Roadie to access the YAML metadata files that Backstage needs to operate. Learn more about the [permissions required](/docs/integrations/github-app-permissions/).
+Installing the GitHub App will allow Roadie to access the YAML metadata files that Backstage needs to operate. Learn more about the [permissions required](/docs/plugins/github-app-permissions/).
```

Comment:
Iain is actually in the process of reverting this change in #650. 

We need to keep the URLs static because people may have linked to them from internal documentation or other websites.

Reply:
Yup, saw that. I'll wait for your tabs refactor to go in before merging this.

Reply:
You should be free to work away now.